sync/metrics.go

```diff
@@ -11,40 +12,152 @@ import (
 var meter = otel.Meter("header/sync")
 
 type metrics struct {
-	totalSynced      atomic.Int64
-	totalSyncedGauge metric.Float64ObservableGauge
+	totalSynced     atomic.Int64
+	totalSyncedInst metric.Int64ObservableGauge
+
+	syncLoopStarted       metric.Int64Counter
```

Comment:
I don't think this is valuable.

Reply:
It is valuable to track how often you need to sync. Basically, it tracks how often your node falls out of sync.